scripts/aws/find_instance.sh

```diff
@@ -28,6 +28,10 @@ elif [ $ins_name == "spr-guilty-search" ]; then
     max_ins_num=5
     ins_type="c7i.16xlarge"
```

Comment:
suggest to change this `ins_type` directly

Reply:
> suggest to change this `ins_type` directly

Does it mean that I have to:
1. Delete the older spr-guilty-search c7i.16xlarge instances 
2. And also rename the c7i metal to spr-guilty-search?

And the c7i metal are new instances, how can I init them for guilty commit search use?

Reply:
yes, we need delete all old spr-guilty-search c7i.16xlarge instances. We don't need manually init new spr-guilty-search c7i metal instance, it will be created by guilty commit search job and init it directly.

Reply:
> yes, we need delete all old spr-guilty-search c7i.16xlarge instances. We don't need manually init new spr-guilty-search c7i metal instance, it will be created by guilty commit search job and init it directly.

Done